common/reviews/api/debug-certificate-generator.api.md

```diff
@@ -0,0 +1,48 @@
+## API Report File for "@microsoft/debug-certificate-manager"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export class CertificateStore {
+    // (undocumented)
+    certificateData: string | undefined;
+    // (undocumented)
+    readonly certificatePath: string;
+    // (undocumented)
+    static readonly instance: CertificateStore;
+    // (undocumented)
+    keyData: string | undefined;
+    }
+
+// @public
+export function ensureCertificate(canGenerateNewCertificate: boolean, parentTask: ILogging): ICertificate;
+
+// @public (undocumented)
+export interface ICertificate {
+    // (undocumented)
+    pemCertificate: string | undefined;
+    // (undocumented)
+    pemKey: string | undefined;
+}
+
+// @public (undocumented)
+export interface ILogging {
+    // (undocumented)
+    log: (string: any) => void;
+    // (undocumented)
+    logError: (string: any) => void;
+    // (undocumented)
+    logVerbose: (string: any) => void;
+    // (undocumented)
+    logWarning: (string: any) => void;
+}
```

Comment:
Let's just use `Terminal` here.

Reply:
Are you saying I should get a `Terminal` passed in, or that I should create my own `ConsoleTerminalProvider` for these functions. Or something else?

Reply:
Let's pass in a `Terminal` instance.

Reply:
The only example I could find of using Terminal with a GulpTask was in gulp-core-build-typescript. So I extracted the GCBTerminalProvider to an export in @microsoft/gulp-core-build and I'm using that to create the Terminal instance in @microsoft/gulp-core-build-serve. Let me know if I should do this differently.